app/views/IndexPage.scala.html

```diff
@@ -18,7 +18,7 @@
 
 @()(implicit request: Request[_], messages: Messages)
 
-@layout(pageTitle = titleNoForm("emcs-tfe-change-destination-frontend"), showBackLink = false) {
+@layout(pageTitle = titleNoForm("emcs-tfe-change-destination-frontend"), maybeShowActiveTrader = None, showBackLink = false) {
```

Comment:
The user is technically authenticated on this page (according to the controller) so it should be possible to have the banner showing?

Reply:
OK will have a look 👀